### PR TITLE
[5.0] Tinymce Source Code Editor shortcut

### PR DESCRIPTION
**Pull request by brianteeman**
This simple PR adds a keyboard shortcut to open the source code editor when in tinymce

On Windows the key is ALT-U
On Mac the key is OPTION-U

### Testing Instructions
This is a js change so either test with a prebuilt package or apply the pr and then `npm run build:js`

### Actual result BEFORE applying this Pull Request
Nothing

### Expected result AFTER applying this Pull Request
Pressing the key opens the code editor

### Link to documentations
None - yet - needs documentation for the entire code editor plugin to be written AND link to that documentation to be added to tinymce.

### CODE REVIEW

**Review by wilsonge — APPROVED** [review]

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 48478f9b70745e8e9074b4d9fc1cf306f7cdc351<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39644">issues.joomla.org/tracker/joomla-cms/39644</a>.</sub>

**Comment by sandewt** [comment]
> On Windows the key is ALT-U

You mean instead of clicking the 'Toggle Editor' in 'Articles: Edit' mode ?
In that case nothing happens in Windows.

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 48478f9b70745e8e9074b4d9fc1cf306f7cdc351<br /><br />Tested on WINDOWS and NOT on a MAC)
Browsers: Firefox and Edge
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39644">issues.joomla.org/tracker/joomla-cms/39644</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39644">issues.joomla.org/tracker/joomla-cms/39644</a>.</sub>

File: build/media_source/plg_editors_tinymce/js/plugins/highlighter/plugin.es6.js [hunk 1]
```diff
@@ -82,4 +82,5 @@ window.tinymce.PluginManager.add('highlightPlus', (editor, url) => {
     onAction: showSourceEditor,
     context: 'tools',
   });
+  editor.addShortcut('Alt+U', 'Opens the code editor', showSourceEditor);
```
Comment (laoneo) [inline build/media_source/plg_editors_tinymce/js/plugins/highlighter/plugin.es6.js:85]: Where is this string visible, must it be translated?

Comment (brianteeman) [reply build/media_source/plg_editors_tinymce/js/plugins/highlighter/plugin.es6.js:85]: if the tinymce help button is enabled the press Alt+0

**Comment by HLeithner** [comment]
thanks

**Pull request merged**